qutebrowser/config/configdata.py

```diff
@@ -680,6 +685,23 @@ def data(readonly=False):
              "* `{host}`: The host of the current web page.\n"
              "* `{backend}`: Either 'webkit' or 'webengine'"),
 
+            ('title-format-pinned',
+             SettingValue(typ.FormatString(
+                 fields=['perc', 'perc_raw', 'title', 'title_sep', 'index',
+                         'id', 'scroll_pos', 'host'], none_ok=True),
+                 '{index}'),
+             "The format to use for the pinned tab title."
+             " The following placeholders are defined:\n\n"
```

Comment:
I think we can simply say "The same placeholders like for `title-format` are defined." or so.